lib/parser/association-list-parser.ts

```diff
@@ -42,7 +42,7 @@ export class AssociationListParser extends ParserBase {
           association.formalPart = this.extractReads(association, formalTokens, true);
           associationTokens = actualTokens;
         }
-        if (associationTokens[0].getLText() !== 'open') {
+        if (associationTokens[0]?.getLText() !== 'open') {
           association.actualIfInput = this.extractReads(association, associationTokens);
           if (type === 'port') {
             association.actualIfOutput = this.extractReadsOrWrite(association, associationTokens);
```

Comment:
Do you have an example file for this? This should not happen. Only if there is an named association with empty actual?
We could check for associationToken.length > 0 in L43 and throw a ParserError there then?

Reply:
Ah yes, that seems to be the problem. If I am reading the standard correctly, that is not valid vhdl and a parser error would be ok. However,`vsim` parses empty actuals perfectly fine...